README.md

```diff
@@ -118,50 +201,61 @@ Follow the [chisel-template README](https://github.com/freechipsproject/chisel-t
 
 If you insist on setting up your own project from scratch, your project needs to depend on both the chisel3-plugin (Scalac plugin) and the chisel3 library.
 For example, in SBT this could be expressed as:
+
 ```scala
 // build.sbt
-scalaVersion := "2.13.7"
-addCompilerPlugin("edu.berkeley.cs" % "chisel3-plugin" % "3.5.0" cross CrossVersion.full)
-libraryDependencies += "edu.berkeley.cs" %% "chisel3" % "3.5.0"
-// We also recommend using chiseltest for writing unit tests 
+scalaVersion := "2.13.8"
+addCompilerPlugin("edu.berkeley.cs" % "chisel3-plugin" % "3.5.3" cross CrossVersion.full)
+libraryDependencies += "edu.berkeley.cs" %% "chisel3" % "3.5.3"
+// We also recommend using chiseltest for writing unit tests
 libraryDependencies += "edu.berkeley.cs" %% "chiseltest" % "0.5.0" % "test"
 ```
+
 ### Guide For New Contributors
+
 If you are trying to make a contribution to this project, please read [CONTRIBUTING.md](https://github.com/Burnleydev1/chisel3/blob/recent_PR/CONTRIBUTING.md)
 
 ### Design Verification
 
 These simulation-based verification tools are available for Chisel:
-- [**iotesters**](https://github.com/freechipsproject/chisel-testers), specifically [PeekPokeTester](https://github.com/freechipsproject/chisel-testers/wiki/Using%20the%20PeekPokeTester), provides constructs (`peek`, `poke`, `expect`) similar to a non-synthesizable Verilog testbench.
-- [**testers2**](https://github.com/ucb-bar/chisel-testers2) is an in-development replacement for PeekPokeTester, providing the same base constructs but with a streamlined interface and concurrency support with `fork` and `join`.
 
+* [**testers2**](https://github.com/ucb-bar/chisel-testers2) is the batteries-included testing and formal verification library for Chisel-based RTL designs and a replacement for the former PeekPokeTester, providing the same base constructs but with a streamlined interface and concurrency support with `fork` and `join` with internal and Verilator integration for simulations.
+* [**iotesters**](https://github.com/freechipsproject/chisel-testers), specifically [PeekPokeTester](https://github.com/freechipsproject/chisel-testers/wiki/Using%20the%20PeekPokeTester), is the previous-generation test framework for Chisel providing constructs (`peek`, `poke`, `expect`) similar to a non-synthesizable Verilog testbench.
```

Comment:
```suggestion
* [**iotesters**](https://github.com/freechipsproject/chisel-testers) (deprecated), specifically [PeekPokeTester](https://github.com/freechipsproject/chisel-testers/wiki/Using%20the%20PeekPokeTester), is the previous-generation test framework for Chisel providing constructs (`peek`, `poke`, `expect`) similar to a non-synthesizable Verilog testbench.
```
I would call out that this is deprecated.

Reply:
Why don't we just completely remove it? Old users will know the URL and new users should better not try to start using `iotesters` since it might not even be released with the next Chisel version.

Reply:
Yeah sounds good. You'll get loud and annoying deprecation warnings if you use it in 3.5 anyway.

Reply:
```suggestion
```